subiquity/ui/views/tests/test_identity.py

```diff
@@ -41,6 +66,27 @@ def test_done_enabled_when_valid(self):
         view_helpers.enter_data(view.form, valid_data)
         self.assertTrue(view.form.done_btn.enabled)
 
+    async def test_username_validation_system_reserved(self):
```

Comment:
This doesn't do anything unless you change the class to inherit from unittest.IsolatedAsyncioTestCase

Reply:
Indeed, but just inheriting my test class from IsolatedAsyncioTestCase breaks TestMessageHub. I'm fighting for a while with that break without getting a clue of what's going on. Every call to `run_coro`  inside tests_pubsub.py throws `RuntimeError: There is no current event loop in thread 'MainThread'`. I'm not quite sure why testing the identity view disturbs MessageHub tests.

Reply:
Hah! I had no idea about all of that.